Mathlib/Probability/Cdf.lean

```diff
@@ -69,8 +69,8 @@ lemma tendsto_cdf_atBot : Tendsto (cdf μ) atBot (𝓝 0) := tendsto_condCdf_atB
 /-- The cdf tends to 1 at +∞. -/
 lemma tendsto_cdf_atTop : Tendsto (cdf μ) atTop (𝓝 1) := tendsto_condCdf_atTop _ _
 
-set_option synthInstance.maxHeartbeats 25000 in
 lemma ofReal_cdf [IsProbabilityMeasure μ] (x : ℝ) : ENNReal.ofReal (cdf μ x) = μ (Iic x) := by
+  have := IsProbabilityMeasure.toIsFiniteMeasure (Measure.prod (Measure.dirac ()) μ)
```

Comment:
`set_option synthInstance.maxHeartbeats 50000 in` also works here, but `25000` is not enough anymore.

Reply:
really crazy that it affects something here.